packages/core/src/Stores/notification-store.js

```diff
@@ -291,12 +291,17 @@ export default class NotificationStore extends BaseStore {
                     is_10k_withdrawal_limit_reached &&
                     (needs_verification.includes('document') || document?.status !== 'verified');
                 const needs_poi = is_10k_withdrawal_limit_reached && identity?.status !== 'verified';
+                const needs_poinc =
+                    needs_verification.includes('income') && ['rejected', 'none'].includes(income?.status);
+                const poinc_upload_limited = needs_verification.includes('income') && income?.status === 'locked';
```

Comment:
```suggestion
                const poinc_upload_limited = needs_verification.includes('income') && income?.status === income_status_codes.locked;
```

Reply:
According to our discussion, we can move status dose for income and other features in separate PR.